rosbot_description/urdf/components/vl53lox.urdf.xacro

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <!-- vl53lox defining macro -->
+  <xacro:macro name="vl53lox" params="xyz z prefix parent">
```

Comment:
I don't think that name `z` for this parameter is suiting. I would prefer something like `yaw_degrees`. Even better would be to just pass `rpy` just as `xyz` - it will be more flexible, similar to how we do it in other places and just simpler.